src/extension.ts

```diff
@@ -39,16 +45,34 @@ class PSCompletionProvider implements CompletionItemProvider {
 }
 
 export async function activate(context: ExtensionContext) {
-  let registerCompletionProvider = languages.registerCompletionItemProvider(
-    'javascript',
-    new PSCompletionProvider(snippets as [Snippet]),
-    '*'
-  )
+  async function workspaceFoldersChange(folders: WorkspaceFolder[]) {
+    if (folders) {
```

Comment:
Prettier options could be read from User's Settings, for instance. Don't you think it's wrong not to allow the extension to be used, then? Especially since we can provide defaults.

I'm thinking opened files with no workspace (folder).

Reply:
Agree.